_pages/keyboards/gergo.md

```diff
@@ -24,3 +29,5 @@ Here is a printable default layout for the first few days of getting used to Ger
 ## Gadgets
 Gergo comes suited for gadgets to be added to the keyboard. Currently a 1u Trackball is avalible and others
 are on the way! Modify your rules.mk to enable/disable specfic gadgets.
+
+Or if you're impaitent, build your own using the open source footprint avalible [on Github!](http://github.com/germ/germ-footprints). 
```

Comment:
`s/impaitent/impatient/`
`s/avalible/available/`

```suggestion
Or if you're impatient, build your own using the open source footprint available [on Github!](http://github.com/germ/germ-footprints). 
```

Also, I think this would read better if it were combined with the previous paragraph. (Basically, delete the two line breaks between this line and Line 31.